content/en/docs/Updating/_index.md

```diff
@@ -6,26 +6,49 @@ description: >
  Keeping the theme up to date.
 ---
 
-We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the theme submodule yourself as follows:
+We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). 
+If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the Docsy theme
+yourself. 
 
-1. In your local copy of your project, run:
+Updating Docsy means that your site will build using the latest version of Docsy at `HEAD` and include 
+all the new commits or changes that have been merged since the time when you first added the Docsy submodule, 
+or last updated. If you have any modified content like a Docsy [customiztion](/docs/adding-content/lookandfeel/), those 
```

Comment:
This won't affect any modifications you may have made to [override the Docsy look and feel](/docs/adding-content/lookandfeel) in your own project, as these don't modify the theme itself. For details of what has changed...

Reply:
(just to make it clear that the modifications in question are overrides in your project, rather than changes to the theme itself)

Reply:
Thanks @LisaFC! FYI, if you click 
![image](https://user-images.githubusercontent.com/11762685/58900258-af4f7980-86b3-11e9-9cc1-5f52aa0f90a1.png) in the comment box, you can suggest an inline edit.